pkgs/tools/networking/picosnitch/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, psutil, dbus-python, requests }:
+
+buildPythonPackage rec {
+  pname = "picosnitch";
+  version = "0.9.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "e56aef62f0f9a48a346f02a96362b0e655cc754ebacaf58cb1632e602463d365";
+  };
+
+  propagatedBuildInputs = [ psutil dbus-python requests ];
+
+  pythonImportsCheck = [ "picosnitch" ];
+
+  meta = with lib; {
+    description = "An extremely simple, reliable, and lightweight program for linux to help protect your privacy";
```

```suggestion
    description = "Tool for Linux to help protect your privacy";
```

It's an opinionated changes but a lot of adjectives do no help :wink: